lib/util/log.sh

```diff
@@ -51,49 +59,38 @@ Log() {
   local logger
   local logged
 
-  if [[ ! -z "$subject" ]]
-  then
-    if [[ ! -z "${__oo__logScopeOutputs["$scope/$callingFunction/$subject"]}" ]]
-    then
+  if [[ ! -z "$subject" ]]; then
+    if [[ ! -z "${__oo__logScopeOutputs["$scope/$callingFunction/$subject"]}" ]]; then
       loggerList="${__oo__logScopeOutputs["$scope/$callingFunction/$subject"]}"
-    elif [[ ! -z "${__oo__logScopeOutputs["$scope/$subject"]}" ]]
-    then
+    elif [[ ! -z "${__oo__logScopeOutputs["$scope/$subject"]}" ]]; then
       loggerList="${__oo__logScopeOutputs["$scope/$subject"]}"
-    elif [[ ! -z "${__oo__logScopeOutputs["$subject"]}" ]]
-    then
+    elif [[ ! -z "${__oo__logScopeOutputs["$subject"]}" ]]; then
       loggerList="${__oo__logScopeOutputs["$subject"]}"
     fi
 
-    loggers=( ${loggerList//;/ } )
-    for logger in "${loggers[@]}"
-    do
+    loggers=(${loggerList//;/ })
+    for logger in "${loggers[@]}"; do
       subject="${subject:-LOG}" Log::Using "$logger" "$@"
       logged=true
     done
   fi
 
-  if [[ ! -z "${__oo__logScopeOutputs["$scope/$callingFunction"]}" ]]
-  then
-    if [[ -z $logged ]] || [[ ${__oo__logDisabledFilter["$scope/$callingFunction"]} == true || ${__oo__logDisabledFilter["$scope"]} == true ]]
-    then
+  if [[ ! -z "${__oo__logScopeOutputs["$scope/$callingFunction"]}" ]]; then
+    if [[ -z $logged ]] || [[ ${__oo__logDisabledFilter["$scope/$callingFunction"]} == true || ${__oo__logDisabledFilter["$scope"]} == true ]]; then
       loggerList="${__oo__logScopeOutputs["$scope/$callingFunction"]}"
-      loggers=( ${loggerList//;/ } )
-      for logger in "${loggers[@]}"
-      do
-          subject="${subject:-LOG}" Log::Using "$logger" "$@"
-          logged=true
+      loggers=(${loggerList//;/ })
+      for logger in "${loggers[@]}"; do
+        subject="${subject:-LOG}" Log::Using "$logger" "$@"
+        logged=true
       done
     fi
   fi
 
-  if [[ ! -z "${__oo__logScopeOutputs["$scope"]}" ]]
-  then
-    if [[ -z $logged ]] || [[ ${__oo__logDisabledFilter["$scope"]} == true ]]
-    then
+  if [[ ! -z "${__oo__logScopeOutputs["$scope"]}" ]]; then
```

Comment:
The repository uses syntax with a new line, so I'd prefer to keep it consistent.